koswat/strategies/strategy_reinforcement_type_costs.py

```diff
@@ -9,7 +9,9 @@
 class StrategyReinforcementTypeCosts:
     reinforcement_type: type[ReinforcementProfileProtocol]
     base_costs: float = 0.0
+    base_costs_with_surtax: float = 0.0  # Not needed yet
```

Comment:
Then why adding it? Inline comments are disliked by Sonarcloud, place it above.

Reply:
Adjusted